recipes-soletta/soletta/soletta_git.bb

```diff
@@ -38,16 +44,124 @@ FILES_${PN}-dbg = " \
+                ${datadir}/soletta/flow/descriptions/keyboard.json \
+                ${datadir}/soletta/flow/descriptions/file.json \
                 ${includedir}/soletta/* \
                 ${libdir}/pkgconfig/soletta.pc \
-                ${libdir}/soletta/modules/flow/* \
+                ${libdir}/soletta/modules/flow/twitter.so \
+                ${libdir}/soletta/modules/flow/test.so \
+                ${libdir}/soletta/modules/flow/si114x.so \
+                ${libdir}/soletta/modules/flow/power-supply.so \
+                ${libdir}/soletta/modules/flow/piezo-speaker.so \
+                ${libdir}/soletta/modules/flow/oauth.so \
+                ${libdir}/soletta/modules/flow/network.so \
+                ${libdir}/soletta/modules/flow/max31855.so \
+                ${libdir}/soletta/modules/flow/iio.so \
+                ${libdir}/soletta/modules/flow/http-server.so \
+                ${libdir}/soletta/modules/flow/hc-sr04.so \
+                ${libdir}/soletta/modules/flow/flower-power.so \
+                ${libdir}/soletta/modules/flow/compass.so \
+                ${libdir}/soletta/modules/flow/update.so \
+                ${libdir}/soletta/modules/flow/unix-socket.so \
+                ${libdir}/soletta/modules/flow/udev.so \
+                ${libdir}/soletta/modules/flow/thingspeak.so \
+                ${libdir}/soletta/modules/flow/stts751.so \
+                ${libdir}/soletta/modules/flow/servo-motor.so \
+                ${libdir}/soletta/modules/flow/process.so \
+                ${libdir}/soletta/modules/flow/persistence.so \
+                ${libdir}/soletta/modules/flow/oic.so \
+                ${libdir}/soletta/modules/flow/magnetometer.so \
+                ${libdir}/soletta/modules/flow/location.so \
+                ${libdir}/soletta/modules/flow/led-strip.so \
+                ${libdir}/soletta/modules/flow/keyboard.so \
+                ${libdir}/soletta/modules/flow/json.so \
+                ${libdir}/soletta/modules/flow/jhd1313m1.so \
+                ${libdir}/soletta/modules/flow/http-client.so \
+                ${libdir}/soletta/modules/flow/gyroscope.so \
+                ${libdir}/soletta/modules/flow/grove.so \
+                ${libdir}/soletta/modules/flow/form.so \
+                ${libdir}/soletta/modules/flow/format.so \
+                ${libdir}/soletta/modules/flow/file.so \
+                ${libdir}/soletta/modules/flow/evdev.so \
+                ${libdir}/soletta/modules/flow/calamari.so \
+                ${libdir}/soletta/modules/flow/am2315.so \
+                ${libdir}/soletta/modules/flow/accelerometer.so \
+                ${libdir}/soletta/modules/flow/mqtt.so \
                 ${libdir}/soletta/modules/pin-mux/* \
                 ${libdir}/soletta/modules/linux-micro/* \
                 ${libdir}/soletta/modules/flow-metatype/* \
                 ${sysconfdir}/modules-load.d/* \
 "
 
+ALLOW_EMPTY_${PN}-dev = "1"
+RDEPENDS_${PN}-dev = " \
```

Comment:
Yes it should be flow-gtk.

You are right, bitbake should detect automatically if ```gtk.so```. 
I will retest everything without ```RDEPENDS``` and see if it will work